app/decorators/proposal_decorator.rb

```diff
@@ -54,4 +54,12 @@ def step_text_for_user(key, user)
     scope = [:decorators, :steps, klass]
     I18n.t(key, scope: scope)
   end
+
+  def self.csv_headers
```

Comment:
these methods could be good candidates for the Fields objects created in https://github.com/18F/C2/pull/920 once that is merged.